src/util.ts

```diff
@@ -35,3 +35,14 @@ export const shortString = (st : string, n: number) => st.length > n ? st.substr
 export function isCarRequest(request : Request) {
   return request.request.headers.some((header) => header.name.toLowerCase() == 'content-type' && header.value == CAR.contentType)
 }
+
+export function formatError(error: any): string {
+  if (error === null || error === undefined) {
+    return "Error: null or undefined";
+  }
```

Comment:
**suggestion:** I think we can leave this out and it'll work slightly better. `JSON.stringify(null)` is `"null"`, and `String(undefined)` is `"undefined"`.